src/renderers/dom/test/ReactTestUtilsEntry.js

```diff
@@ -437,45 +437,39 @@ var ReactTestUtils = {
 /**
  * Exports:
  *
- * - `ReactTestUtils.Simulate.click(Element/ReactDOMComponent)`
- * - `ReactTestUtils.Simulate.mouseMove(Element/ReactDOMComponent)`
- * - `ReactTestUtils.Simulate.change(Element/ReactDOMComponent)`
+ * - `ReactTestUtils.Simulate.click(Element)`
+ * - `ReactTestUtils.Simulate.mouseMove(Element)`
+ * - `ReactTestUtils.Simulate.change(Element)`
  * - ... (All keys from event plugin `eventTypes` objects)
  */
 function makeSimulator(eventType) {
-  return function(domComponentOrNode, eventData) {
-    var node;
+  return function(domNode, eventData) {
     invariant(
-      !React.isValidElement(domComponentOrNode),
-      'TestUtils.Simulate expects a component instance and not a ReactElement.' +
+      !React.isValidElement(domNode),
+      'TestUtils.Simulate expects a component instance and not a ReactElement. ' +
```

Comment:
In fact let's make the wording identical to the new invariant but with "received a React element" instead of "received a component instance".